### DISCUSSION TITLE
moving demo UIs from readme to the Wiki

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#244] [Ideas]
I have begun moving the existing demo UIs to the Wiki. 

[Here's](https://github.com/UNC-Project-Open-AAC/OS-DPI/wiki/Visual-scene-display) the VSD page.

Questions for the larger group -
- is this existing information sufficient for understanding as a new person entering the demo space?
- is the screen recording helpful or no?
- links work for everyone?
- any other information that should be included?

**Comment by gbishop** [comment]
This is great! Everything works for me. I can't answer the first question. I like the screen recording. 

For anything tricky, we could write a few paragraphs describing how it works.

**Comment by aetkie** [comment]
Additional UIs have been added (contact-like system and mandarin-english board). What other demos would anyone like to have available in the Wiki?
